Specs/DateTime/English/DateTimeModel.json

```diff
@@ -6600,5 +6600,155 @@
+      "ReferenceDateTime": "2018-08-31T12:00:00"
+    },
+    "NotSupported": "javascript, python, java",
+    "Results": [
+      {
+        "Text": "year greater than 2012",
+        "Start": 21,
+        "End": 42,
+        "TypeName": "datetimeV2.daterange",
+        "Resolution": {
+          "values": [
+            {
+              "timex": "2012",
+              "Mod": "after",
+              "type": "daterange",
+              "start": "2012-12-31"
+            }
+          ]
+        }
+      }
+    ]
+  },
+  {
+    "Input": "I want sales for year 2012 or later",
+    "Context": {
+      "ReferenceDateTime": "2018-08-31T12:00:00"
+    },
+    "NotSupported": "javascript, python, java",
+    "Results": [
+      {
+        "Text": "year 2012 or later",
+        "Start": 17,
+        "End": 34,
+        "TypeName": "datetimeV2.daterange",
+        "Resolution": {
+          "values": [
+            {
+              "timex": "2012",
+              "Mod": "since",
+              "type": "daterange",
+              "start": "2012-01-01"
+            }
+          ]
+        }
+      }
+    ]
+  },
+  {
+    "Input": "How about year 2016 and greater",
+    "Context": {
+      "ReferenceDateTime": "2018-08-31T12:00:00"
+    },
+    "NotSupported": "javascript, python, java",
+    "Results": [
+      {
+        "Text": "year 2016 and greater",
+        "Start": 10,
+        "End": 30,
+        "TypeName": "datetimeV2.daterange",
+        "Resolution": {
+          "values": [
+            {
+              "timex": "2016",
+              "Mod": "since",
+              "type": "daterange",
+              "start": "2016-01-01"
+            }
+          ]
+        }
+      }
+    ]
+  },
+  {
+    "Input": "You can only leave on 1/1/2016 and later",
+    "Context": {
+      "ReferenceDateTime": "2018-08-31T12:00:00"
+    },
+    "NotSupported": "javascript, python, java",
+    "Results": [
+      {
+        "Text": "1/1/2016 and later",
+        "Start": 22,
+        "End": 39,
+        "TypeName": "datetimeV2.daterange",
+        "Resolution": {
+          "values": [
+            {
+              "timex": "2016-01-01",
+              "Mod": "since",
+              "type": "daterange",
+              "start": "2016-01-01"
+            }
+          ]
+        }
+      }
+    ]
+  },
+  {
+    "Input": "You can only leave on 1/1/2016 and later",
+    "Context": {
+      "ReferenceDateTime": "2018-08-31T12:00:00"
+    },
+    "NotSupported": "javascript, python, java",
+    "Results": [
+      {
+        "Text": "1/1/2016 and later",
+        "Start": 22,
+        "End": 39,
+        "TypeName": "datetimeV2.daterange",
+        "Resolution": {
+          "values": [
+            {
+              "timex": "2016-01-01",
+              "Mod": "since",
+              "type": "daterange",
+              "start": "2016-01-01"
+            }
+          ]
+        }
+      }
+    ]
+  },
+  {
+    "Input": "You can only leave on 1/1/2016 and after",

```

Comment:
I'd prefer if these "and after" cases can only work at the end of a sentence.
Can you add a case that shows one sentence with two uses of dates. For example:
"I can only leave on 1/1/2016 and after my work item is done"
and
"I can only leave on 1/1/2016 and after 6PM"

Reply:
Thanks for catching these failed cases.
Fixed.